data/ayatana-indicator-bluetooth.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Why weren't these moved with `git mv` and then modified, so that it shows up as rename with only the necessary changes in the diff? It makes some of these changes harder to follow, so better practice is to use `git mv` to rename the files then edit them, when this is necessary.

Reply:
This was not intended to be a pull request, but a direct merge addressing the move to cmake and the switch to Ayatana namespace, where all the issues would be ironed out afterwards (such as the bug we inherited from upstream, mentioned in the comment).